packages/suite-base/src/players/constants.ts

```diff
@@ -29,3 +29,27 @@ export const FIELD_SIZE_BY_PRIMITIVE: Record<string, number> = {
   string: 20, // we don't know the length upfront, assume a fixed length
 };
 export const MAX_NUM_FAST_PROPERTIES = 1020;
+
+// Capabilities that are not shared by all players.
+export const PlayerCapabilities = {
```

Comment:
Please, rename it using refactoring (usually F2 in vscode)
```suggestion
export const PLAYER_CAPABILITIES = {
```

Reply:
Suggestion implemented.